check_tweets.py

```diff
@@ -0,0 +1,54 @@
+import mwclient
+import tweepy
+from datetime import date, timedelta
+import time
+from dotenv import load_dotenv
+import os
+import re
+
+load_dotenv()
+bearertoken = os.getenv("bearertoken")
+
+checkfrom = date.today() - timedelta(days=4)
+
+def main():
+	client = tweepy.Client(bearer_token=bearertoken)
+
+	site = mwclient.Site('lol.fandom.com', path='/')
```

Comment:
```
    credentials = AuthCredentials(user_file='me')
    site = EsportsClient('lol', credentials=credentials)
```

the library will help prompt you to set up a login session file locally with your bot password (you can set this up at `Special:BotPasswords` on the wiki)